app/components/error_list/connection_error.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React from 'react';
+import {injectIntl} from 'react-intl';
+import {
+    StyleSheet,
+    View,
+    Text
+} from 'react-native';
+
+const style = StyleSheet.create({
+    container: {
+        alignSelf: 'stretch',
+        alignItems: 'center',
+        justifyContent: 'center',
+        padding: 10
+    }
+});
+
+function ConnectionError(intl) {
+    return (
+        <View style={style.container}>
+            <Text>
+                {
+                  intl.formatMessage({
```

Comment:
you don't need to add intl here if you use <FormattedText> component found in app/components/formatted_text